labs.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: "Labs"
+permalink: labs
+---
+
+Here are some academic groups specifically doing research around Sold. If you are interested in being listed send an email to info@solidproject.org with a description of the research project and link to the website. 
+
+#[MIT](https://www.mit.edu) - [CSAIL Solid Social Linked Data](https://www.csail.mit.edu/research/solid-social-linked-data) 
```

Comment:
Let's perhaps not have "Social Linked Data" on the website to avoid confusion. I know it's historically correct, but people will think it is still actual.

Reply:
Also, the group is DIG.

Reply:
Sounds good, just edited. Let me know if more edits needed.